### ISSUE TITLE
Bookmarks are duplicated / created automatically

### ISSUE THREAD

**Issue by harry-g** [Morikko/sync-tab-groups#42]
If the "save groups in bookmarks automatically" option is enabled (which is default):
* bookmarks are additionally created outside of SyncTabGroupsFolder
* bookmarks are duplicated, not overwritten (I think each time the tab is activated and the 30s are over)
** this way for some pages I got 80 or 100 times the same bookmark
* turning off "save groups in bookmarks automatically" solves the issue
This seriously "spams" bookmarks, as it bookmarks every single page visited in main bookmarks folder.
Until solved, I would propose tu set the default to disabled (maybe even after it is solved).

Observed in Firefox 57.0 under Windows 10 (Version 10.0.15063 Build 15063)

**Comment by Morikko** [comment]
Sure, I totally agree. I have disabled it by default for new user.
Already users have to disable it themselves.
I have also added a note for warning user.

I am facing many issues with this and I don't want to focus on this first as it will be time consuming. I prefer focus on other features more important first.

But sure, I will rewrite the function to be lighter, safer and cleaner. Sorry for the disagreement.
